tests/test_indexer.py

```diff
@@ -327,6 +347,65 @@ def get_search_results(cls, query, expected_hit_count):
                 time.sleep(0.5)
 
 
+class BundleBuilder:
+    def __init__(self, bundle_id=None, bundle_version=None):
+        self.bundle_id = bundle_id if bundle_id else str(uuid.uuid4())
+        self.bundle_version = bundle_version if bundle_version else self._get_version()
+        self.bundle_manifest = {
+            BundleMetadata.FORMAT: BundleMetadata.FILE_FORMAT_VERSION,
+            BundleMetadata.VERSION: self.bundle_version,
+            BundleMetadata.FILES: [],
+            BundleMetadata.CREATOR_UID: "0"
+        }
+
+    def get_bundle_id(self):
+        return f'{self.bundle_id}.{self.bundle_version}'
+
+    def add_file(self, bucket_name, name, indexed, file_id):
+        # Add the existing file to the bundle manifest
+        s3 = boto3.resource("s3")
+        file_manifest_string = s3.Object(bucket_name=bucket_name,
+                                         key=f'files/{file_id}').get()['Body'].read().decode('utf-8')
+        file_manifest = json.loads(file_manifest_string, encoding="utf-8")
+        file_uuid, _, file_version = file_id.partition(".")
```

Comment:
This doesn't seem to work due to two variables on the left side and three substrings from splitting on '.'
```
file_uuid, file_version = '6011caee-ebde-4b2b-82f1-bb75d303fa41.2017-08-07T150159.427496Z'.split(".")
Traceback (most recent call last):
  File "<input>", line 1, in <module>
ValueError: too many values to unpack (expected 2)
```
Although it works if `maxsplit=1` is specified:
```
file_uuid, file_version = '6011caee-ebde-4b2b-82f1-bb75d303fa41.2017-08-07T150159.427496Z'.split(".", maxsplit=1)
```
@kislyuk used partition in https://github.com/HumanCellAtlas/data-store/pull/291 so I used it here as well.